include/morphio/morphology.h

```diff
@@ -82,6 +82,12 @@ class Morphology
      **/
     const Points& points() const;
 
+    /**
+     * Returns a list with offsets to access data of a specific section in the points
+     * and diameters arrays
+     **/
+    const std::vector<int> sectionOffsets() const;
```

Comment:
The output of section_offsets looks correct to me.
Just one small question here,  would it be better to use uint32_t for offset  so that it is consistent with the type of section_id?

Reply:
Good point, I've changed that.